### PR TITLE
[sysdig-deploy] Update subcharts and readme clarifications

### PR DESCRIPTION
**Pull request by achandras**
## What this PR does / why we need it:

Pull in versions with latest patches and update the README with some missing info

## Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)
- [x] PR only contains changes for one chart

### CODE REVIEW

File: charts/sysdig-deploy/README.md [hunk 1]
```diff
@@ -118,23 +119,24 @@ helm upgrade -n sysdig-agent sysdig sysdig/sysdig-deploy -f values.sysdig.yaml
...
-| `kspmCollector.apiEndpoint`      | kspmCollector apiEndpoint                                         | `""`      |
+| Parameter                               | Description                                                                                                             | Default   |
+| --------------------------------------- | ----------------------------------------------------------------------------------------------------------------------- | --------- |
+| `global.clusterConfig.name`             | Identifier for this cluster                                                                                             | `""`      |
+| `global.sysdig.accessKey`               | Sysdig Agent Access Key                                                                                                 | `""`      |
+| `global.sysdig.accessKeySecret`         | The name of a Kubernetes secret containing an 'access-key' entry.                                                       | `""`      |
+| `global.sysdig.region`                  | The SaaS region for these agents                                                                                        | `"us1"`   |
```
Comment (achandras) [inline charts/sysdig-deploy/README.md]: Sorry for the huge formatting change 😞 this line is the only "real" change

Comment (yashgiri) [reply charts/sysdig-deploy/README.md]: I see the same variable repeated twice lines 127,128. is this correct?

Comment (achandras) [reply charts/sysdig-deploy/README.md]: Definitely an error, thanks! The line *below* that should be the only real change 😄

**Review by lilx1ao — APPROVED** [review]

**Review by yashgiri — APPROVED** [review]
Thank you for the changes! the readme looks even better now.

**Review by yashgiri — APPROVED** [review]

**Pull request merged**